ext/hermann/rdcrc32.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * \file rdcrc32.h
+ * Functions and types for CRC checks.
+ *
+ * Generated on Tue May  8 17:36:59 2012,
+ * by pycrc v0.7.10, http://www.tty1.net/pycrc/
```

Comment:
This is a header file from librdkafka. The code is included in `librdkafka.a` (https://github.com/edenhill/librdkafka/blob/850737abd2c7ae0544716794c4a7e836f9b732f9/src/rdcrc32.c#L119) and this header file (https://github.com/braintree/hermann/blob/stable-partition/ext/hermann/rdcrc32.h#L94). The functions weren't exposed in the `rdkafka.h` file, for some reason. This is fixed on master of librdkafka, which has `rd_kafka_msg_partitioner_consistent `: https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L554